### PR TITLE
Don't look for RGW pods in independent clusters

### LINKED ISSUE #2601: Skip RGW object bucket tests on irrelevant platforms
Title.

At the moment, the tests run regardless of platform - even on ones where RGW isn't deployed out of the box.

### PR DESCRIPTION
**Pull request by MeridianExplorer**
At the moment, upon every instantiation of an MCG object, an RGW pod check is run.
However, this check fails at clusters running in independent mode.
This PR is supposed to fix this by not looking for the pods when running on an independent cluster.

Closes #2601

### CODE REVIEW

File: ocs_ci/ocs/resources/mcg.py [hunk 1]
```diff
@@ -136,7 +137,10 @@ def __init__(self, *args, **kwargs):
             pods = pod.get_pods_having_label(label=constants.RGW_APP_LABEL, namespace=self.namespace)
             assert len(pods) == 0, 'RGW pod should not exist on AWS platform'
 
-        elif config.ENV_DATA.get('platform') in constants.ON_PREM_PLATFORMS:
+        elif (
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/mcg.py:140]: @nehaberry @rachael-george @sidhant-agrawal is it always the case that RGW pod is not deployed in independent mode clusters?

Comment (rachael-george) [reply ocs_ci/ocs/resources/mcg.py:140]: Yes, the RGW exists in the RHCS cluster and does not run as a pod on the OCS nodes

Comment (ebenahar) [reply ocs_ci/ocs/resources/mcg.py:140]: @MeridianExplorer we have discussed this offline, I think we should also add validation for making sure there is no RGW pod running on the OCS node with external clusters (independent mode)

File: ocs_ci/ocs/resources/mcg.py [hunk 2]
```diff
@@ -131,9 +132,14 @@ def __init__(self, *args, **kwargs):
                 aws_access_key_id=self.aws_access_key_id,
                 aws_secret_access_key=self.aws_access_key
             )
+
+        if (
+            config.ENV_DATA['platform'].lower() in constants.CLOUD_PLATFORMS
+            or storagecluster_independent_check() is True
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/mcg.py:138]: no need for `is True`

File: ocs_ci/ocs/resources/mcg.py [hunk 3]
```diff
@@ -131,9 +132,14 @@ def __init__(self, *args, **kwargs):
...
+        ):
             logger.info('Checking whether RGW pod is not present on AWS platform')
             pods = pod.get_pods_having_label(label=constants.RGW_APP_LABEL, namespace=self.namespace)
-            assert len(pods) == 0, 'RGW pod should not exist on AWS platform'
+            assert len(pods) == 0, 'RGW pods should not exist in the current platform/cluster'
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/mcg.py:142]:
```suggestion
            assert not pods, 'RGW pods should not exist in the current platform/cluster'
```

**Comment by MeridianExplorer** [comment]
Verified on an AWS cluster - 
```
tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[3-S3] 
-------------------------------------------------------------------------------------------------------------------------------------- live log setup ---------------------------------------------------------------------------------------------------------------------------------------
16:12:36 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get pods -o name
16:12:38 - MainThread - ocs_ci.ocs.utils - INFO - pod name match found appending rook-ceph-tools-7ffdc44c-5mrvj
16:12:38 - MainThread - ocs_ci.ocs.utils - INFO - Ceph toolbox already exists, skipping
16:12:38 - MainThread - tests.conftest - INFO - All logs located at /tmp/ocs-ci-logs-1596373872
16:12:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: /home/meridian/repos/ocs-ci/bin/oc version --client
16:12:39 - MainThread - ocs_ci.utility.utils - INFO - OpenShift Client version: Client Version: 4.5.0-0.nightly-2020-07-18-024505

16:12:39 - MainThread - ocs_ci.ocs.version - INFO - collecting ocs version
16:12:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get clusterversion version -o yaml
16:12:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get namespace  -o yaml
16:12:46 - MainThread - ocs_ci.ocs.version - INFO - found storage namespaces ['openshift-cluster-storage-operator', 'openshift-kube-storage-version-migrator', 'openshift-kube-storage-version-migrator-operator', 'openshift-storage']
16:12:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-cluster-storage-operator --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get pod  -n openshift-cluster-storage-operator -o yaml
16:12:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-kube-storage-version-migrator --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get pod  -n openshift-kube-storage-version-migrator -o yaml
16:12:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-kube-storage-version-migrator-operator --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get pod  -n openshift-kube-storage-version-migrator-operator -o yaml
16:12:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get pod  -n openshift-storage -o yaml
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - ClusterVersion .spec.channel: stable-4.5
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - ClusterVersion .status.desired.version: 4.5.0-0.nightly-2020-08-01-204100
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - ClusterVersion .status.desired.image: registry.svc.ci.openshift.org/ocp/release@sha256:9c4998332afbad3df94c77c11df43c69e54e065adfc53df230e41eb3530dd5a3
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - storage namespace openshift-cluster-storage-operator
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:c840fc827070c20258f26ec339422a5359e0f2dd566d4b3a5c6add78347c5748 {'quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:c840fc827070c20258f26ec339422a5359e0f2dd566d4b3a5c6add78347c5748'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:5a725a9d03a85e583f62d6a86e91193d25ab7429881bffd04513eb8f32ac30cb {'quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:5a725a9d03a85e583f62d6a86e91193d25ab7429881bffd04513eb8f32ac30cb'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d879e3dc7515d56799911c3a0208b31f19daef3653c14ca959aa100aaa4482b1 {'quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d879e3dc7515d56799911c3a0208b31f19daef3653c14ca959aa100aaa4482b1'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - storage namespace openshift-kube-storage-version-migrator
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6173259225165fdabe3692d5ef8fbaa6a2b5001a91177165fac3c316c76fe381 {'quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6173259225165fdabe3692d5ef8fbaa6a2b5001a91177165fac3c316c76fe381'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - storage namespace openshift-kube-storage-version-migrator-operator
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:caf6493434690e790f94d26dd0fef59671645fb0e93259712ce4027bb720d2eb {'quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:caf6493434690e790f94d26dd0fef59671645fb0e93259712ce4027bb720d2eb'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - storage namespace openshift-storage
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/rhceph-dev/cephcsi@sha256:6a9808c12a72c2d3ab0900986d3b554eaa6a37532786e592d2478bcbda813a5c {'quay.io/rhceph-dev/cephcsi@sha256:6a9808c12a72c2d3ab0900986d3b554eaa6a37532786e592d2478bcbda813a5c'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:5ed44407414538adcba19f3780be37193f1b5a8789fbf3a4e7ed65c7693f1b66 {'registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:3be38cacdce4218756d927fff577b32fb2cf0e168a26a3713d92208d6df98bc0'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:fc97bc7790703299ee19d9c4353699e049773babbfe74d0b7691ad8536cf250a {'registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:86a4e81a39f5e9682ecfcb02c2af236fd4a05679f82d6e538dc5209fc6b18093'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:f04a827f92c355f3f486957518bcf282fd7b9546ecdc275d04e8389f6290e712 {'registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:0fd907aec05ffa8d46af15fdd1d23aa7fb0cc1283dfad31b086e34551e10295e'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:bc1b07743216f68f0d292e7ed7607b2ce04c7f81d453a09cecc4cc3571eb19e5 {'registry.redhat.io/openshift4/ose-csi-external-resizer-rhel7@sha256:bc1b07743216f68f0d292e7ed7607b2ce04c7f81d453a09cecc4cc3571eb19e5'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/rhceph-dev/mcg-core@sha256:241d9693c1166273ae386e51c317035785eaa24c31d0341e2a72a55d089d2558 {'quay.io/rhceph-dev/mcg-core@sha256:241d9693c1166273ae386e51c317035785eaa24c31d0341e2a72a55d089d2558'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:ba74027bb4b244df0b0823ee29aa927d729da33edaa20ebdf51a2430cc6b4e95 {'registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:ba74027bb4b244df0b0823ee29aa927d729da33edaa20ebdf51a2430cc6b4e95'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/rhceph-dev/mcg-operator@sha256:4acdf2149b23a54ccf8ab2b4f3a69b671ddfd86e936091eae957348ff435dc97 {'quay.io/rhceph-dev/mcg-operator@sha256:4acdf2149b23a54ccf8ab2b4f3a69b671ddfd86e936091eae957348ff435dc97'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/rhceph-dev/ocs-operator@sha256:5c91b968e192fa1b5861cb76cbc57720f1079278a2da0b4dd5b1c5c595b058ac {'quay.io/rhceph-dev/ocs-operator@sha256:5c91b968e192fa1b5861cb76cbc57720f1079278a2da0b4dd5b1c5c595b058ac'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/rhceph-dev/rhceph@sha256:df67e134c9037707118a8533670b2f77647355a4af2fdc1247d79679cb6bb676 {'quay.io/rhceph-dev/rhceph@sha256:df67e134c9037707118a8533670b2f77647355a4af2fdc1247d79679cb6bb676'}
16:13:24 - MainThread - ocs_ci.ocs.version - INFO - image quay.io/rhceph-dev/rook-ceph@sha256:0272d0020093fa5bec67cc75f82b32d310b4250deb3cd679905c5106a3cb3363 {'quay.io/rhceph-dev/rook-ceph@sha256:0272d0020093fa5bec67cc75f82b32d310b4250deb3cd679905c5106a3cb3363'}
16:13:24 - MainThread - tests.conftest - INFO - human readable ocs version info written into /home/meridian/repos/beneli-aws/ocs_version.2020-08-02T16:13:24.500827
16:13:24 - MainThread - ocs_ci.utility.utils - INFO - testrun_name: OCS4-5-Downstream-OCP4-5-AWS-IPI-1AZ-RHCOS-3M-3W-acceptance
16:13:24 - MainThread - ocs_ci.utility.utils - INFO - The rook directory /home/meridian/repos/ocs-ci/data/rook already exists, ocs-ci will skip the clone of rook repository.
16:13:24 - MainThread - ocs_ci.utility.utils - INFO - Fetching latest changes from rook repository.
16:13:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git fetch --all
16:13:26 - MainThread - ocs_ci.utility.utils - INFO - Checkout rook repository to specific branch: master
16:13:26 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git checkout master
16:13:26 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Already on 'master'

16:13:26 - MainThread - ocs_ci.utility.utils - INFO - Reset branch: master with latest changes
16:13:26 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git reset --hard origin/master
16:13:26 - MainThread - tests.conftest - INFO - Checking for Ceph Health OK 
16:13:26 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
16:13:27 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
16:13:28 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-7ffdc44c-5mrvj -- ceph health
16:13:31 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
16:13:31 - MainThread - tests.conftest - INFO - Ceph health check passed at setup
16:13:31 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get Pod  -n openshift-storage --selector=noobaa-operator=deployment -o yaml
16:13:32 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get Pod  -n openshift-storage --selector=noobaa-core=noobaa -o yaml
16:13:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get Pod noobaa-operator-68cd9d45d5-cjcq2 -n openshift-storage -o yaml
16:13:36 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig rsh noobaa-operator-68cd9d45d5-cjcq2 bash -c "md5sum /usr/local/bin/noobaa-operator"
16:13:39 - MainThread - ocs_ci.ocs.resources.pod - INFO - md5sum of file /usr/local/bin/noobaa-operator: 50afa475979557bffbd270259161e48b
16:13:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-ingress-operator --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get secret router-ca -n openshift-ingress-operator -o yaml
16:13:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get noobaa  -n openshift-storage -o yaml
16:13:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get secret noobaa-admin -n openshift-storage -o yaml
16:13:59 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/mcg.py - INFO - Checking whether RGW pod is not present on AWS platform
16:13:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/meridian/repos/beneli-aws/auth/kubeconfig get Pod  -n openshift-storage --selector=app=rook-ceph-rgw -o yaml
--------------------------------------------------------------------------------------------------------------------------------------- live log call ---------------------------------------------------------------------------------------------------------------------------------------
16:14:00 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Creating bucket: s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf
16:14:01 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Waiting for s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf to be healthy
16:14:01 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/mcg.py - INFO - Requesting information about bucket s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf
16:14:02 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf status is OPTIMAL
16:14:02 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf is healthy
16:14:02 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Creating bucket: s3-bucket-fd807c49418247fbafe1c28ce2e776d1
16:14:02 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Waiting for s3-bucket-fd807c49418247fbafe1c28ce2e776d1 to be healthy
16:14:02 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/mcg.py - INFO - Requesting information about bucket s3-bucket-fd807c49418247fbafe1c28ce2e776d1
16:14:03 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-fd807c49418247fbafe1c28ce2e776d1 status is OPTIMAL
16:14:03 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-fd807c49418247fbafe1c28ce2e776d1 is healthy
16:14:03 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Creating bucket: s3-bucket-8386814eeca24546965e5267bfe1eb9f
16:14:03 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Waiting for s3-bucket-8386814eeca24546965e5267bfe1eb9f to be healthy
16:14:03 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/mcg.py - INFO - Requesting information about bucket s3-bucket-8386814eeca24546965e5267bfe1eb9f
16:14:04 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-8386814eeca24546965e5267bfe1eb9f status is OPTIMAL
16:14:04 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-8386814eeca24546965e5267bfe1eb9f is healthy
PASSED                                                                                                                                                                                                                                                                                [ 33%]
------------------------------------------------------------------------------------------------------------------------------------- live log teardown -------------------------------------------------------------------------------------------------------------------------------------
16:14:04 - MainThread - tests.conftest - INFO - Cleaning up bucket s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf
16:14:04 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Deleting bucket: s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf
16:14:05 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Verifying deletion of s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf
16:14:05 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-5b5c07edb14c4659b03de7d3db3c4ddf was deleted successfuly
16:14:05 - MainThread - tests.conftest - INFO - Cleaning up bucket s3-bucket-fd807c49418247fbafe1c28ce2e776d1
16:14:05 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Deleting bucket: s3-bucket-fd807c49418247fbafe1c28ce2e776d1
16:14:05 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Verifying deletion of s3-bucket-fd807c49418247fbafe1c28ce2e776d1
16:14:05 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-fd807c49418247fbafe1c28ce2e776d1 was deleted successfuly
16:14:05 - MainThread - tests.conftest - INFO - Cleaning up bucket s3-bucket-8386814eeca24546965e5267bfe1eb9f
16:14:05 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Deleting bucket: s3-bucket-8386814eeca24546965e5267bfe1eb9f
16:14:06 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - Verifying deletion of s3-bucket-8386814eeca24546965e5267bfe1eb9f
16:14:06 - MainThread - /home/meridian/repos/ocs-ci/ocs_ci/ocs/resources/objectbucket.py - INFO - s3-bucket-8386814eeca24546965e5267bfe1eb9f was deleted successfuly
16:14:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
16:14:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
16:14:08 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-7ffdc44c-5mrvj -- ceph health
16:14:10 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
16:14:10 - MainThread - tests.conftest - INFO - Ceph health check passed at teardown
```

**Pull request merged**